entgql/schema.go

```diff
@@ -727,6 +727,12 @@ func (e *schemaGenerator) mapScalar(gqlType string, f *gen.Field, ant *Annotatio
 			// will generate an enum named "TodoStatus".
 			scalar = gqlType + scalar
 		}
+		if f.Type.RType != nil {
+			switch f.Type.RType.Kind {
+			case reflect.Slice, reflect.Array:
+				scalar = "[" + scalar + "!]"
```

Comment:
The "!" (not null) needs to be inferred from the type. Unlike `T`, `*T` is nullable.

Reply:
@giautm, your thoughts on it? Users can always fallback to the `Type()` option.

Reply:
> The "!" (not null) needs to be inferred from the type. Unlike `T`, `*T` is nullable.

agree.

Reply:
How to fix?